microsoft/testsuites/hpc/infinibandsuite.py

```diff
@@ -571,6 +580,12 @@ def verify_mvapich_mpi(self, environment: Environment, log: Logger) -> None:
             raise SkippedException(err)
 
         run_in_parallel([server_ib.install_mvapich_mpi, client_ib.install_mvapich_mpi])
+        test_names = ["IMB-MPI1", "IMB-RMA", "IMB-NBC"]
```

Comment:
Add `if` block for Mariner.

Reply:
This is moved from line no 596 to reuse for mariner. I will add if block after this as this is required for other tests also.